docs/general/ad-filtering/create-own-filters.md

```diff
@@ -2218,6 +2244,16 @@ Rules with `$removeparam` modifier are supported by AdGuard for Windows, Mac and
 
 :::
 
+:::caution Restrictions
+
+AdGuard Browser Extension MV3 has some limitations:
+
+- Allowlist rules are not supported.
+- Regexp, negation and allow rules are not supported.
+- Rules with the same matching condition are combined into one, but only within the scope of one static filter or within the scope of all dynamic rules (custom filters and user rules).
```

Comment:
It's impossible to understand what it means

Reply:
@SeverAnna we can add examples here